roles/openjdk/meta/main.yml

```diff
@@ -14,5 +14,5 @@ galaxy_info:
         - 16.04
         - 18.04
 
-dependencies:
-  - {role: common}
+dependencies: []
+#   - {role: common}
```

Comment:
suggestion: Remove the commented-out line.

Reply:
good idea, thanks

Reply:
addressed in c24073f756c948537a461764eb8b41b029186f49